pkgs/applications/science/math/wxmaxima/default.nix

```diff
@@ -11,20 +11,22 @@
 
 stdenv.mkDerivation rec {
   pname = "wxmaxima";
-  version = "22.05.0";
+  version = "22.12.0";
 
   src = fetchFromGitHub {
     owner = "wxMaxima-developers";
     repo = "wxmaxima";
     rev = "Version-${version}";
-    sha256 = "sha256-pcKnEjJmvMXCBpjtOSLyl4I0x3fjh0os9Sdp39I2Re0=";
+    sha256 = "sha256-RT6y4M6LQD1fXJcjtdSXnDmoJvv160g2asdV4WtTcok=";
   };
 
   buildInputs = [
     wxGTK
     maxima
     # So it won't embed svg files into headers.
     gnome.adwaita-icon-theme
+    # So it won't crash under Sway.
+    glib
```

Comment:
`glib` is missing as an argument, causing the build to fail.

Reply:
Oh my! Fixed.